docs/swagger/amm-liquidity-routes-cosmos.yml

```diff
@@ -0,0 +1,85 @@
+paths:
```

Comment:
please don't override the default docs for these endpoints. currently, the Swagger addition causes duplicate entries in local docs when users go to localhost:8080
<img width="1245" alt="Screen Shot 2023-12-12 at 5 38 31 PM" src="https://github.com/hummingbot/gateway/assets/1795030/15db968b-16bc-49c9-aee6-a137ca5ec2b7">


if you want to show users how to use cosmos-specific endpoints, I suggest doing so on the doc page for this in the https://github.com/hummingbot/hummingbot-site repo.

Reply:
hi @fengtality, we have a PR planned for the /hummingbot-site repo with pages for osmosis dex, osmosis chain, update to cosmos chain, and the change to index, as described in the contribution guidelines - just havent done the PR yet, its coming up.